components/indicators/IndicatorProgressBar.js

```diff
@@ -433,6 +434,7 @@ function IndicatorProgressBar(props) {
                   value={formatValue(startValue, i18n.language, minPrecision)}
                   unit={unit}
                   locale={i18n.language}
+                  negative={readableColor(startColor) === '#ffffff'}
```

Comment:
This feels like it could be prone to error (for example if `readableColor` is updated and starts returning `'#FFF'` instead), could we pass a `color` prop to ValueGroup instead? No need to do this now though

Reply:
Or we could define both the background colour and text colour in the theme